<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm2</class>
 <widget class="QMainWindow" name="MainForm2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="MiangridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="fileActionOpen"/>
    <addaction name="actionnew"/>
    <addaction name="actionclos"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addWinAction"/>
  </widget>
  <action name="fileActionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionnew">
   <property name="text">
    <string>new </string>
   </property>
  </action>
  <action name="actionclos">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="addWinAction">
   <property name="text">
    <string>添加窗体</string>
   </property>
   <property name="toolTip">
    <string>添加窗体</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
